### ISSUE TITLE
Bug/Request: Single line alignment of VALUE Expressions with BASE/FOR

### ISSUE THREAD

**Issue by stockbal** [SAP/abap-cleaner#170]
Hi,

in an an earlier version - I think before 1.8.0 - small short VALUE expressions that used BASE or FOR were usually kept in one line. Now they are always put into 2 lines.

The following snippet:

```abap
result = VALUE #( BASE result ( NEW zcl_sat_os_subp_method_std( ) ) ).
result = VALUE #( BASE result ( comp = 4 ) ).
lt_cds_range = VALUE #( FOR cds IN io_node_helper->mt_cds_views ( sign = 'I' option = 'EQ' low = cds-name ) ).
```

... is formatted into this in the current version:

```abap
result = VALUE #( BASE result
                  ( NEW zcl_sat_os_subp_method_std( ) ) ).
result = VALUE #( BASE result
                  ( comp = 4 ) ).
lt_cds_range = VALUE #( FOR cds IN io_node_helper->mt_cds_views 
                        ( sign = 'I' option = 'EQ' low = cds-name ) ).
```

If this change is because of some clean code rule, I would like the option that such short assignment are kept in one line.
If it's a bug, then please fix it.

Thanks and Regards,
Ludwig

**Comment by jmgrassau** [comment]
Hi Ludwig, 

you're right, this was previously kept on one line, but unintentionally (see #154): the option "Table rows: Keep multiple components on single line" did more than it was meant to do. But I do see your point – maybe we just need a dedicated option for these cases…

Kind regards,
Jörg-Michael

**Comment by stockbal** [comment]
Hi Jörg-Michael,

a new option sounds good 😊.

Kind regards,
Ludwig

**Comment by jmgrassau** [comment]
Hi Ludwig, 

with the next release, you will find the following new option in the rule "Align parameters and components":

![image](https://github.com/SAP/abap-cleaner/assets/79639298/3ac73899-b350-4c40-b43d-921291d06018)

For "Never", you get:

![image](https://github.com/SAP/abap-cleaner/assets/79639298/8acd8719-affc-41cd-a7e5-e540535e3b87)

For "Always" or "If maximum line length A is observed", the code on the left-hand side is preserved. This is then similar to the behavior before version 1.8.0, but a bit stricter, requiring line length A (default: 120 chars) to be observed instead of line length B (default: 160 chars). 

Kind regards,
Jörg-Michael

**Comment by stockbal** [comment]
Hi Jörg-Michael,

thanks a lot 😊👍🏻. 

Kind regards,
Ludwig

**Comment by jmgrassau** [comment]
Hi Ludwig, 

thanks for opening this issue – the new option is now available with version 1.13.0, which was just released!

Kind regards,
Jörg-Michael
